templates/web/base/admin/report-category.html

```diff
@@ -1,5 +1,8 @@
 [%~ BLOCK category_option ~%]
-<option value="[% cat.category | html %]"[% ' selected' IF problem.category == cat.category %]>[% cat.category_display | html %]</option>
+[% IF cat.disable_form_field AND NOT cat.body.get_cobrand_handler.staff_can_assign_reports_to_disabled_categories %]
```

Comment:
Whilst this is the standard way of marking a category as disabled (and I think Bath probably does use it), any extra data can actually do it with its `disable_form` flag, so I think we'd need to have a new method to check if a category gets disabled, here.

I don't think we need to worry about handler, because staff will always be on their cobrand, so cobrand.staff_can_assign... would probably do?